python/lsst/ts/mtrotator/config_schema.py

```diff
@@ -0,0 +1,52 @@
+# This file is part of ts_mtrotator.
+#
+# Developed for the Rubin Observatory Telescope and Site System.
```

Comment:
I think we agreed on `Vera Rubin Observatory` but I'll leave it up to you to decide if you want to update this and the other files.

Reply:
I don't know if we have an official standard. I think this is close enough for now.